src/containers/LoginForm.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { inject, observer } from 'mobx-react/native';
+import { action, observable } from 'mobx';
+import LoginForm from '../components/LoginForm';
+import { login } from '../authApi';
+
+// todo use mobx-react-form instead?
+@inject('authStore')
+@inject('errorStore')
+@observer
+class LoginFormContainer extends React.Component {
+  @observable userName = '';
```

Comment:
You should either use a store or keep this in state. This breaks a lot of the react component lifecycle by using observables in containers... or did you see people using this patten?